### ISSUE TITLE
[1.10.2] Damaged items desyncs transmute table

### ISSUE THREAD

**Issue by Tencao** [sinkillerj/ProjectE#1437]
So I've had an issue which I have reported via irc where damaged items would only appear client side and be completely invisible server side, causing them both to be desynced.  This can lead to weird stuff like emc going missing, ghost items, etc. To give you a visual, look at the image below that causes the bug.

https://cdn.discordapp.com/attachments/257169622338371585/258713923744694272/Przechwytywanie.PNG

In an attempt to fix this myself, I've tracked down the bug and my fix seems to solve it, however it may not be 100% ideal nor might it be the best since my understanding of the code is limited, but to cut it short I basically made the transmute slot read the damaged item as if it was fully repaired, there for instead of showing a damaged item in the slots (that causes the bug) it now shows the full one, and the emc requirement of the full one matches the item fully repaired, not the current one. 

http://image.prntscr.com/image/96d20fc3312544b4bb9089261bfea939.png

The code I changed for this is this part here:
https://github.com/sinkillerj/ProjectE/blob/master/src/main/java/moze_intel/projecte/gameObjs/container/inventory/TransmutationInventory.java#L160-L168

I've had to reorder certain parts to make it actually fit properly, but this is the fix I used. 
http://paste.ee/r/BzY6d

Basically I moved the lockCopy first then made the copy fully repaired if damaged, then get the lockCopy's emc instead of the raw items emc.
